rag/vector_databases/retrievers.py

```diff
@@ -0,0 +1,14 @@
+from rag.vector_databases import init_vdb
+from rag.utils.configurables import (
+    retriever_searchtype_configurable,
+    retriever_searchkwargs_configurable,
+)
+
+chatbot_retriever = (
```

Comment:
Turn into function, error happens when trying to call vector database locally as still searching for url.